config.go

```diff
@@ -10,110 +10,151 @@ const (
 	if err != nil {
 		log.Errorf("error setting config value: %v", err)
 	}
 }
 
-func (lw *LibWallet) SetLongConfigValueForKey(key string, value int64) {
-	err := lw.SaveUserConfigValue(key, value)
+func (mw *MultiWallet) SetInt32ConfigValueForKey(key string, value int32) {
+	err := mw.SaveUserConfigValue(key, value)
 	if err != nil {
 		log.Errorf("error setting config value: %v", err)
 	}
 }
 
-func (lw *LibWallet) SetStringConfigValueForKey(key, value string) {
-	err := lw.SaveUserConfigValue(key, value)
+func (mw *MultiWallet) SetLongConfigValueForKey(key string, value int64) {
+	err := mw.SaveUserConfigValue(key, value)
 	if err != nil {
 		log.Errorf("error setting config value: %v", err)
 	}
 }
 
-func (lw *LibWallet) ReadBoolConfigValueForKey(key string) (valueOut bool) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
+func (mw *MultiWallet) SetStringConfigValueForKey(key, value string) {
+	err := mw.SaveUserConfigValue(key, value)
+	if err != nil {
+		log.Errorf("error setting config value: %v", err)
+	}
+}
+
+func (mw *MultiWallet) ReadBoolConfigValueForKey(key string, defaultValue bool) (valueOut bool) {
+	err := mw.ReadUserConfigValue(key, &valueOut)
+	if err != nil {
+		if err == storm.ErrNotFound {
+			valueOut = defaultValue
+			return
+		}
+		log.Errorf("error reading config value: %v", err)
+	}
+
+	return
+}
+
+func (mw *MultiWallet) ReadDoubleConfigValueForKey(key string, defaultValue float64) (valueOut float64) {
+	err := mw.ReadUserConfigValue(key, &valueOut)
 	if err != nil {
+		if err == storm.ErrNotFound {
+			valueOut = defaultValue
+			return
+		}
 		log.Errorf("error reading config value: %v", err)
 	}
 	return
 }
 
-func (lw *LibWallet) ReadDoubleConfigValueForKey(key string) (valueOut float64) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
+func (mw *MultiWallet) ReadIntConfigValueForKey(key string, defaultValue int) (valueOut int) {
+	err := mw.ReadUserConfigValue(key, &valueOut)
 	if err != nil {
+		if err == storm.ErrNotFound {
+			valueOut = defaultValue
+			return
+		}
 		log.Errorf("error reading config value: %v", err)
 	}
 	return
 }
 
-func (lw *LibWallet) ReadIntConfigValueForKey(key string) (valueOut int) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
+func (mw *MultiWallet) ReadInt32ConfigValueForKey(key string, defaultValue int32) (valueOut int32) {
+	err := mw.ReadUserConfigValue(key, &valueOut)
 	if err != nil {
+		if err == storm.ErrNotFound {
+			valueOut = defaultValue
+			return
+		}
 		log.Errorf("error reading config value: %v", err)
 	}
 	return
 }
 
-func (lw *LibWallet) ReadLongConfigValueForKey(key string) (valueOut int64) {
-	err := lw.ReadUserConfigValue(key, &valueOut)
+func (mw *MultiWallet) ReadLongConfigValueForKey(key string, defaultValue int64) (valueOut int64) {
+	err := mw.ReadUserConfigValue(key, &valueOut)
 	if err != nil {
+		if err == storm.ErrNotFound {
+			valueOut = defaultValue
+			return
+		}
 		log.Errorf("error reading config value: %v", err)
```

Comment:
If as suggested above, you add a `defaultValue` property to the `mw.ReadUserConfigValue` method, then we can simply do `err := mw.ReadUserConfigValue(key, &valueOut, defaultValue)`. Also, log all `error reading config value` messages from the `mw.ReadUserConfigValue` method.